src/mlpack/methods/ann/ffn_impl.hpp

```diff
@@ -559,6 +559,8 @@ void FFN<
   // Reset the network parameters with the given initialization rule.
   NetworkInitialization<InitializationRuleType> networkInit(initializeRule);
   networkInit.Initialize(network.Network(), parameters);
+  // Override the weight matrix.
```

Comment:
```suggestion
  // Override the weight matrix if necessary.
```
This may not actually do anything, so, it would be good to point that out in the comments. :+1: